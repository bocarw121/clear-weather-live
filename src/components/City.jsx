import React from "react";
import { StyleSheet, Text } from "react-native";

import colors from "../constants/colors";

const City = ({ city }) => {
  return <Text style={styles.location}>{city}</Text>;
};

export default City;

const styles = StyleSheet.create({
  location: {
    color: colors.white,
    fontSize: 34,
    marginTop: 15,
    marginBottom: 10,
    textAlign: "center",
  },
});
