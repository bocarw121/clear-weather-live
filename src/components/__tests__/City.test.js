import React from "react";

import { render } from "@testing-library/react-native";

import City from "../City";

it("Should render the City", () => {
  const screen = render(<City />);

  expect(screen).toMatchSnapshot();
});
