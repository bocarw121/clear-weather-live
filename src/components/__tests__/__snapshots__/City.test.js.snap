// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the City 1`] = `
<Text
  style={
    Object {
      "color": "#fff",
      "fontSize": 34,
      "marginBottom": 15,
      "marginTop": 15,
      "textAlign": "center",
    }
  }
/>
`;
